Marlin/src/sd/cardreader.cpp

```diff
@@ -540,6 +540,7 @@ void CardReader::manage_media() {
   }
   else {
     TERN_(HAS_SD_DETECT, release()); // Card is released
+    TERN_(HAS_WIRED_LCD, ui.return_to_status());  // Force SPI/I2C LCD models to return to status screen
```

Comment:
It might make more sense to just have "wired" LCDs do this only when they want to, in response to the call to `ui.media_changed(old_stat, stat)` below, instead of here. The idea of `media_changed` is to allow for the `CardReader` class to simply broadcast an event and let the LCD code handle it, rather than moving logic based on "intimate knowledge" about specific LCDs or LCD types into the `CardReader` class.

Reply:
I see, that's better